### DISCUSSION TITLE
[Bug]: Updating 'Improve Contrast' setting from the UI writes a quoted instead of unquoted boolean value to config.yml

### DISCUSSION THREAD

**Discussion by neutralvibes** [blakeblackshear/frigate#13543] [Report a Bug] [answered]
### Checklist

- [X] I have updated to the latest available Frigate version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.
- [ ] I have tried reproducing the issue in [incognito mode](https://www.computerworld.com/article/1719851/how-to-go-incognito-in-chrome-firefox-safari-and-edge.html) to rule out problems with any third party extensions or plugins I have installed.

### Describe the problem you are having

Updating the `Improve Contrast` camera setting from the UI updates the entry in config.yml to a quoted instead of unquoted boolean value. literally ['true' |'false'] rather than [true|false].

It is unknown whether it is interpreted correctly by frigate internally, although the setting in the  UI shows the change. The `Config Editor` highlights this as a problem, however allows the file to be saved.

### Steps to reproduce

1. Ensure improve_contrast setting for a camera is unquoted in config.yml
2. Toggle Improve Contrast for same camera in the UI.
3. Observe new setting in config.yml
...

### Version

0.14.1-f4f3cfa

### Frigate config file

```yaml
# Example written
    motion:
      mask: 0.003,0.002,1,0,1,0.478,0,0.466
      threshold: 30
      contour_area: 10
      improve_contrast: 'true'
```

### docker-compose file or Docker CLI command

```yaml
N/A
```

### Relevant Frigate log output

```shell
N/A
```

### Relevant go2rtc log output

```shell
N/A
```

### Operating system

Debian

### Install method

Docker Compose

### Network connection

Wired

### Camera make and model

n/a

### Screenshots of the Frigate UI's System metrics pages

n/a

**Comment by NickM-27** [comment]
This is not a bug, the parser can handle both formats

**Comment by hawkeye217** [comment, marked as answer]
This has been improved in https://github.com/blakeblackshear/frigate/pull/13550. As @NickM-27 said, the parser handles both formats so there is no functionality bug. But my PR should fix the quotes being put around boolean values and the built-in config editor highlighting it as an issue.
